cosmos/cosvm.json

```diff
@@ -0,0 +1,51 @@
+{
+  "rpc": "https://cvm.cosvm.net",
+  "rest": "https://api.cosvm.net",
+  "chainId": "cosvm_323-1",
+  "chainName": "CosVM Network",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/cosvm/chain.png",
+  "stakeCurrency": {
+    "coinDenom": "CVM",
+    "coinMinimalDenom": "ucvm",
+    "coinDecimals": 18,
+    "coinGeckoId": "cvm",
+    "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/cosvm/acosvm.png"
+  },
+  "walletUrl": "https://wallet.keplr.app/chains/cosvm",
+  "walletUrlForStaking": "https://wallet.keplr.app/chains/cosvm",
```

Comment:
@Financiya-Network Hello.
Could you remove `walletUrl`, `walletUrlForStaking` field?

Reply:
do i remove those link from json file ?